addons/l10n_ma/models/res_company.py

```diff
@@ -0,0 +1,20 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+from odoo import api, fields, models
+
+
+class ResCompany(models.Model):
+    _inherit = 'res.company'
+
+    l10n_ma_ice = fields.Char("ICE", help='Common Company Identifier')
+
+class BaseDocumentLayout(models.TransientModel):
+    _inherit = 'base.document.layout'
+
+    @api.model
+    def _default_company_details(self):
+        company_details = super()._default_company_details()
+        if self.env.company.country_code == 'MA':
+            company_details += ("ICE: " + self.env.company.l10n_ma_ice)
```

Comment:
Why not the new line anymore?

Reply:
@william-andre 
I added the new line in the super function actually for that, but it seems like it's a bad thing to do for the reason you mentioned.
The issue I'm trying to fix is the security issue. I tried to use a static string like in the super function but it still raising security issue.

```
@api.model
    def _default_company_details(self):
        company_details = super()._default_company_details()
        if self.env.company.country_code == 'MA':
            address_format = '<br> ICE: %s'
            company_details += Markup(address_format) % self.env.company.l10n_ma_ice
        return company_details
```